package.json

```diff
@@ -8,15 +8,17 @@
     "private::lint.black": "if [[ $PYLINTRC != '.pylintrc' ]]; then black dash tests --exclude metadata_test.py --check; fi",
     "private::lint.flake8": "flake8 --exclude=metadata_test.py dash tests",
     "private::lint.pylint-dash": "PYLINTRC=${PYLINTRC:=.pylintrc37} && pylint dash setup.py --rcfile=$PYLINTRC",
-    "private::lint.pylint-tests": "pylint tests/unit tests/integration -d all -e C0410,C0411,C0412,C0413,W0109",
+    "private::lint.pylint-tests": "PYLINTRC=${PYLINTRC:=.pylintrc37} && pylint tests/unit tests/integration -d all --rcfile=$PYLINTRC",
     "private::lint.renderer": "cd dash-renderer && npm run lint",
+    "private::test.setup-nested": "cd \\@plotly/dash-generator-test-component-nested && npm ci && npm run build && pip install -e .",
+    "private::test.setup-standard": "cd \\@plotly/dash-generator-test-component-standard && npm ci && npm run build && pip install -e .",
```

Comment:
Is your plan - the reason these packages are up in `@plotly` rather than in `tests` - to publish them for use in the R (& Julia) tests?

Reply:
No. Don't plan on publishing them at this point. I simply wanted to isolate them from other concerns. The `package.json` file contains `"private": true` which should prevent accidental publication.